operators/multiclusterobservability/manifests/base/grafana/dash-acm-openshift-virtualization-overview.yaml

```diff
@@ -207,38 +210,30 @@ data:
           "targets": [
             {
               "exemplar": true,
-              "expr": "count (count by (cluster) (csv_succeeded{name=~\".*hyperconverged.*\",cluster=~\"$cluster\"}))",
+              "expr": "sum(count(kubevirt_vm_running_status_last_transition_timestamp_seconds{cluster=~\"$cluster\"}) by (name))",
+              "instant": false,
               "interval": "",
               "legendFormat": "",
               "refId": "A"
             }
           ],
-          "title": "Total Clusters with OpenShift Virtualization",
+          "title": "Total VMs",
```

Comment:
@dockerymick I'm not sure if we should replace the VMs to virtual machines due to the title length. @moadz wdyt?

Reply:
Ok that's valid. I recommend that you write out "virtual machines (VMs)" in the first instance and then throughout the rest of the UI use VMs so that users understand what the acronym means